parkings/models/enforcement_domain.py

```diff
@@ -10,8 +10,7 @@
 class EnforcementDomain(TimestampedModelMixin):
     code = models.CharField(max_length=10, unique=True, verbose_name=_('code'))
     name = models.CharField(max_length=40, verbose_name=_('name'))
-    geom = gis_models.MultiPolygonField(
-        srid=GK25FIN_SRID, verbose_name=_('geometry'))
+    geom = gis_models.MultiPolygonField(srid=GK25FIN_SRID, verbose_name=_('geometry'), default="MULTIPOLYGON EMPTY")
```

Comment:
Maybe the geom field should be nullable?

Then we could easily check, if the geometry of the ED is specified or not. That could affect some business logic later.

Reply:
If I put **null=True**. It breaks the admin view. So had to set **default="MULTIPOLYGON EMPTY"** instead of **null=True**